arrow_pd_parser/pa_pd.py

```diff
@@ -11,6 +11,18 @@ def generate_type_mapper(
     pd_date_type=None,
     pd_timestamp_type=None,
 ):
+    """Specifies the pyarrow data types mapping to corresponding Pandas data types.
+
+    Args:
+        pd_boolean: specifies need to reference new pandas bool type. Defaults to None.
```

Comment:
Could the 'specifies need to reference' part be clearer in this and the next couple of descriptions? They might benefit from more clarity as they seem to be slightly odd arguments (they're basically bools, but structured as None/something rather than True/False).

Maybe something like: 

`pd_integer: if not None, use the new Pandas nullable integer type rather than defaulting to floats.`